transcom/mymove
# INT B-21113 customer address language changes

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21113)

## Summary

The following changes were made on the Customer side in order to make address related language consistent across the MilMove app:

1. "Destination Address" -> "Delivery Address"
2. "Pickup Location" -> "Pickup Address"
3. "Current Address" -> "Pickup Address"
4. "Origin Address" -> "Pickup Address"

### How to test

1. Login as a customer.
2. Create a move and add one of every shipment, on the forms, verify that all form fields and titles use the above language, with no refrerence to "Origin" or "Destination".
3. At the end, view the card for each shipment and verify that each title for the address fields use the above language, example screenshot below.

## Screenshots
![milmovelocal_3000_moves_3767f7f6-6b14-4680-ae9e-529600e60654_review (1)](https://github.com/user-attachments/assets/f84682e3-7a3a-454b-9aac-17ff61d4a5ca)
